data/l1db-ap-pipe-schema.yaml

```diff
@@ -0,0 +1,1681 @@
+# Generated from /home/salnikov/cat/sql/baselineSchema.sql
```

Comment:
I defined two extra columns related to the observing filter name and integer id in the DIASource table. There isn't a CcdVisit table in l1dbproto currently so I need to care around this extra information to compute summary fluxes for DIAObjects. Since I just need to the two columns, is this something I could add to a "extra" schema as you have define for the main l1db class? Do these columns interact the same as the main columns for the afw mappings?

Reply:
Right, just add those columns to "extra" schema, they should behave exactly as normal columns from regular schema.